### DISCUSSION TITLE
How this plugin is working in IOS , can any one explains this?

### DISCUSSION THREAD

**Discussion by sadaqatdev** [jakepurple13/Full-Multiplatform-Compose-Plugin#5] [General]
How this plugin is working in IOS , can any one explains this?

**Comment by jakepurple13** [comment]
I believe it's because it's all skia under the hood. I suggest taking a look at https://github.com/JetBrains/compose-jb to find out more.

**Comment by sadaqatdev** [comment]
if this use skia under the hood so its like flutter and  have performance issue it think ?

**Reply by jakepurple13** [reply]
It does have some performance issues like https://issuetracker.google.com/issues/228995735 that one.
But I also feel like depending on how you write the code, you can minimize the issues.

**Comment by sadaqatdev** [comment]
but this plugin is amazing and in race with flutter

**Comment by sadaqatdev** [comment]
does this plugin is develop by jetbrains?

**Reply by jakepurple13** [reply]
Nope. I made this plugin myself. Used the templates from Jetbrains as a model for what I needed to do.
